.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,7 +5,7 @@ Fixes #\<issue_number>
 ## Before submitting
 
 - [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
-- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
+- [ ] Did you read the [contributor guideline](https://torchmetrics.readthedocs.io/en/latest/generated/CONTRIBUTING.html), Pull Request section?
```

Comment:
let's still link to the markdown in the github repo.

Reply:
```suggestion
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/metrics/blob/master/.github/CONTRIBUTING.md), Pull Request section?
```